gwcli.8

```diff
@@ -163,9 +163,9 @@ The create command will define the client IQN to all gateways within the configu
 The delete command will attempt to remove client IQN from all gateways within the configuration. The client must be logged out, for the delete command to be successful.
 .TP
 .nf
-\fB/iscsi-target/<iqn>/hosts/<client_iqn>/ auth chap=<user>/<pswd> | nochap\fR
+\fB/iscsi-target/<iqn>/hosts/<client_iqn>/ auth chap=<user>/<pswd>\fR
```

Comment:
Is this a limitation of the kernel or gwcli / ceph-iscsi-config? If it's just a limitation in the tooling, perhaps we can just highlight the inconsistent auth issue within the UI (like how it currently shows auth in red if not configured)?

Reply:
It's a limitation in the kernel.

Reply:
Oh yeah, but I think we can also detect and highlight the issue similar to when auth is not configured. I will add that in when I repush.